metadata-db-app/src/cmr/metadata_db/data/const.clj

```diff
@@ -0,0 +1,13 @@
+(ns cmr.metadata-db.data.const
```

Comment:
Is it possible to include these vars in the new data/util.clj namespace instead so all the common data functionality is  in one place?

Reply:
Sure, if you want. My habit is to put constants in a const file, but I don't mind another approach.

Reply:
Done.